protostar/start.py

```diff
@@ -1,37 +1,56 @@
 import asyncio
+import sys
 from pathlib import Path
+from typing import Any
 
 from protostar.cli import (
     ArgumentParserFacade,
     ArgumentValueFromConfigProvider,
     CLIApp,
     MissingRequiredArgumentException,
 )
-from protostar.protostar_cli import ConfigurationProfileCLISchema, ProtostarCLI
+from protostar.composition_root import build_di_container
+from protostar.configuration_profile_cli import ConfigurationProfileCLI
+from protostar.protostar_cli import ProtostarCLI
 from protostar.protostar_exception import UNEXPECTED_PROTOSTAR_ERROR_MSG
+from protostar.protostar_toml.io.protostar_toml_reader import ProtostarTOMLReader
 
 
 def main(script_root: Path):
-    protostar_cli = ProtostarCLI.create(script_root)
+    container = build_di_container(script_root)
+    arg_parser = _build_parser(container.protostar_cli, container.protostar_toml_reader)
+    args = _parse_args(arg_parser)
+    _run_protostar(container.protostar_cli, args, arg_parser)
 
+
+def _build_parser(
```

Comment:
I never seen `_` prefix for functions only for class methods. Can we don't use it?